src/Illuminate/View/Factory.php

```diff
@@ -693,6 +700,67 @@ public function doneRendering()
     }
 
     /**
+     * Add new loop to the stack.
+     *
+     * @param  array|\Countable  $data
+     * @return void
+     */
+    public function addLoop($data)
+    {
+        $length = is_array($data) || $data instanceof \Countable ? count($data) : null;
+
+        $this->loopsStack[] = [
+            'index' => 0,
+            'reverseIndex' => isset($length) ? $length + 1 : null,
+            'size' => $length,
+            'isFirst' => true,
+            'isLast' => isset($length) ? $length == 1 : null,
```

Comment:
But false means it's not the last one, while it can be the last one but we don't know. So maybe null is better.